invite.md

```diff
@@ -0,0 +1,30 @@
+Event: LIDA Visualisation Hackathon 2024
+
+Title: Visualisation of Transport Network Hackathon
+
+Organiser: LIDA Visualisation Programme
+
+Date: Thursday 19th September 2024
```

Comment:
```suggestion
Date: Thursday 3rd October 2024
```

Planning to be in Prague on 19th September, can we move it back a couple of weeks?

Reply:
Absolutely, it seems room booking is not possible for the next year anyways, I better communicate this and maybe leave the room booking for start of Sep?